lessons/topics/01-variables/README.md

```diff
@@ -0,0 +1,20 @@
+# Variables, Scope, Hoisting
+
+In this chapter we will talk about how variables behave in different contexts, how the scope affect the reach of said chunks of data and how we can manage to 
+move the data between blocks and structures
+
+## Previous Knowledge
+
+List or links to this repository for previous Knowledge topics eg.:
+
+* variables Assignment
+* What is a constant? And how do they work?
+* Scope
+* Hoisting
+* Simple Functions
+
```

Comment:
We should add 'Closure' to this list.

The closure concept is really important when we talk about the scope.

Reply:
Closure is more about the  functions behavior so... I would have to dig deeper into how functions work

Reply:
@darthzyklus What do you think about make Closure a new topic that uses this one as previous knowledge?

Reply:
@Hmerin yeah it sounds good, I agree. the closure topic is a little bit more advanced and has more details, it would be better to do that topic in another PR